bindings/nodejs/types/account.ts

```diff
@@ -12,30 +13,39 @@ export interface AccountBalance {
     total: string;
     available: string;
     requiredStorageDeposit: string;
-    nativeTokens: nativeTokensMap;
+    nativeTokens: INativeToken[];
     nfts: string[];
     aliases: string[];
     foundries: string[];
-    potentiallyLockedOutputs: potentiallyLockedOutputsMap;
+    /**
+     * Outputs with multiple unlock conditions and if they can currently be spent or not. If there is a
+     * TimelockUnlockCondition or ExpirationUnlockCondition this can change at any time
+     */
+    potentiallyLockedOutputs: Record<string, boolean>;
 }
 
-export type nativeTokensMap = {
-    [tokenId: string]: number;
-};
-
-export type potentiallyLockedOutputsMap = {
-    [outputId: string]: boolean;
-};
-
 export interface AccountSyncOptions {
+    /**
+     * Specific Bech32 encoded addresses of the account to sync, if addresses are provided,
+     * then `address_start_index` will be ignored
+     */
     addresses?: string[];
+    /**
+     * Address index from which to start syncing addresses. 0 by default, using a higher index will be faster because
+     * addresses with a lower index will be skipped, but could result in a wrong balance for that reason
+     */
     addressStartIndex?: number;
-    automaticOutputConsolidation?: boolean;
+    /**
+     * Usually we skip syncing if it's called within a few seconds, because there can only be new changes every 5
+     * seconds. But if we change the client options, we need to resync, because the new node could be from a nother
+     * network and then we need to check all addresses. This will also ignore `address_start_index` and sync all
+     * addresses. Default: false.
+     */
     forceSyncing?: boolean;
+    /** Checks pending transactions and promotes/reattaches them if necessary.  Default: true. */
     syncPendingTransactions?: boolean;
```

Comment:
```suggestion
    syncTransactions?: boolean;
```

I noticed an inconsistency on the rust side. An alternative is adjusting serde to `syncPendingTransactions` here: https://github.com/iotaledger/wallet.rs/blob/develop/src/account/operations/syncing/options.rs#L24